sites/bizbash.com/server/templates/website-section/venues-destinations.marko

```diff
@@ -65,20 +65,36 @@ $ const { id, alias, name, pageNode } = data;
           <marko-web-website-section-description tag="p" class="page-wrapper__description" obj=section />
         </@section>
       </marko-web-page-wrapper>
-      <div class="row">
-        <for|item| of=blocks>
-          $ const itemHref = `/${item.alias}`;
-          <div class="col-lg-4 mb-block">
+
+<div class="row">
+        $ const children = getAsArray(section, "children.edges").map(({ node }) => node);
```

Comment:
Might be an editor thing, but could you ensure all the indents are consistent :)